### DISCUSSION TITLE
Drafts and published items make problems with pagination

### DISCUSSION THREAD

**Discussion by Feuerhamster** [directus/directus#4895] [Archive]
I'm using Directus as a headless CMS and I only want to show users items with the status "published".
For that I use the filter with `{"status": "published"}` in the API.

But I don't know how to do pagination properly now because the meta field only gives me the counts including all drafts.
Can anyone help me with that?

**Comment by rijkvanzanten** [comment]
The `filter_count` in meta should return the item count with the filters applied 👍🏻
